### PR TITLE
Use multiple IPFS gateways to alleviate rate-limiting

### PR DESCRIPTION
**Pull request by msub2**
Adds in 4 alternative gateways to replace hardcoded ipfs.io links, should help alleviate rate-throttling issues. Includes a small npm script addition for testing purposes. Addresses issue #59

### CODE REVIEW

File: utils/helpers.js [hunk 1]
```diff
@@ -28,6 +28,21 @@ const parseProtocol = uri => {
...
+ * @returns A random public IPFS gateway
+ */
+const getIPFSGateway = () => {
+  const gateways = [
+    'https://gateway.pinata.cloud',
+    'https://cloudflare-ipfs.com',
+    'https://ipfs.fleek.co',
+    'https://dweb.link'
+  ];
+  const rand = Math.floor(Math.random() * (gateways.length));
```
Comment (limbofeather) [inline utils/helpers.js:42]: Not sure if getting the gateways completely randomly makes the most sense here. Can we prioritize the most performant gateways to be used first? Something like 80:20:5:5, or 60:25:10:5?

This looks useful: https://github.com/maxim-saplin/ipfs_gateway_research/blob/main/README.md

File: unity/Assets/ZestySDK/Scripts/Internal/Utils/Utils.cs [hunk 2]
```diff
@@ -54,4 +54,19 @@ public static string ParseProtocol(string uri)
...
+    /// </summary>
+    /// <returns>A random public IPFS gateway</returns>
+    public static string getIPFSGateway() {
+        string[] gateways = {
+            "https://gateway.pinata.cloud",
+            "https://cloudflare-ipfs.com",
+            "https://ipfs.fleek.co",
+            "https://dweb.link"
+        };
+        int rand = UnityEngine.Random.Range(0, gateways.Count() - 1);
```
Comment (limbofeather) [inline unity/Assets/ZestySDK/Scripts/Internal/Utils/Utils.cs:68]: Let’s update gateway selection to be weighted here and not random, too.

File: tests/wonderland/js/package.json [hunk 3]
```diff
@@ -10,6 +10,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@zestymarket/wonderland-sdk": "^1.0.0"
+    "@zestymarket/wonderland-sdk": "^1.4.4"
```
Comment (limbofeather) [inline tests/wonderland/js/package.json:13]: Is this what's necessary for Sorskoot to use beacons?

**Review by limbofeather — APPROVED** [review]
Looks good to me, just one comment about the WLE versioning

File: tests/wonderland/js/package.json (hunk 3 above)
Comment (msub2) [reply tests/wonderland/js/package.json:13]: I just caught it while I was working, it's in one of our test projects, don't know how I didn't notice it until now. Sorskoot should be able to use beacons with the version on NPM right now

Comment (limbofeather) [reply tests/wonderland/js/package.json:13]: No worries, appreciate the fix!

**Pull request merged**